sdks/builds/ios.mk

```diff
@@ -38,23 +30,23 @@ BITCODE_CONFIGURE_FLAGS=--enable-llvm-runtime --with-bitcode=yes
 #  $(3): arch (arm or aarch64)
 #
 # Flags:
-#  ios_$(1)_AC_VARS
-#  ios_$(1)_SYSROOT
-#  ios_$(1)_CONFIGURE_FLAGS
-#  ios_$(1)_CFLAGS
-#  ios_$(1)_CPPFLAGS
-#  ios_$(1)_CXXFLAGS
-#  ios_$(1)_LDFLAGS
-#  ios_$(1)_BITCODE_MARKER
+#  ios-$(1)_AC_VARS
```

Comment:
These are just make variables, why they have to be renamed ? Also, the current naming convention is kinda what automake, and the old ios/android makefiles use.

Reply:
They one do not have to be renamed, it just keep consistency with the `_ios-$(1)_*` variables names which them *needs* to be renamed